test/distributed/_composable/fsdp/test_fully_shard_training.py

```diff
@@ -285,44 +285,11 @@ def test_train_parity_multi_group_eager(self):
             {
                 "reshard_after_forward": [True, False, 2],
                 "device_type": ["cuda"],
-                "offload_policy": [OffloadPolicy()],
-                "delay_after_forward": [False, True],
-                "delay_before_all_gather": [False, True],
-                "delay_before_reduce_scatter": [False, True],
-                "delay_before_optim": [False, True],
-            },
-            self._test_train_parity_multi_group,
-        )
-
-    @skip_if_lt_x_gpu(2)
-    def test_train_parity_multi_group_compile(self):
-        self.run_subtests(
-            {
-                "reshard_after_forward": [True, False],
-                "device_type": ["cuda"],
-                "offload_policy": [OffloadPolicy()],
-                "delay_after_forward": [False, True],
-                "delay_before_all_gather": [False],
-                "delay_before_reduce_scatter": [False],
-                "delay_before_optim": [False, True],
-            },
-            self._test_train_parity_multi_group,
-        )
-
-    @skip_if_lt_x_gpu(2)
-    def test_train_parity_multi_group_cpu_offload_eager(self):
-        """
-        Tests train parity against DDP when using multiple parameter groups for
-        communication and CPU offloading.
-        """
-        self.run_subtests(
-            {
-                "reshard_after_forward": [True],  # save CI time
                 "offload_policy": [
+                    None,
```

Comment:
Is `offload_policy` allowed to be `None`? I thought it was only `OffloadPolicy` or `CPUOffloadPolicy` for now.

Reply:
you're right. I am changing to `OffloadPolicy` now (`None` magically worked but I will respect the type hint)